source/sync/get-started.txt

```diff
@@ -49,10 +46,11 @@ Configure Your Data Model
 Choose an SDK
 ~~~~~~~~~~~~~
 
-The :ref:`{+client-sdks+} <realm-sdks>`, available for multiple languages and
-platforms, include everything you need to work with synced Realms in your client
-application code. You can use multiple SDKs to work with the same set of synced
-data across devices and platforms.
+Realm is an object database optimized for mobile use cases. The
```

Comment:
Just a Q: are we still saying "optimized for mobile use cases?" Isn't C++ more geared toward systems/IoT? I know it's not technically even in Alpha yet, but I wonder if this positioning is changing/will change. Maybe we could try out something like "optimized for performance and agile development on devices."

Reply:
Even though it doesn't _quite_ include IoT, I think it cuts through enough to get the point across: we're using a completely different database on the client because it's optimized for those clients (mobile or lightweight devices).